app/src/main/java/com/yterletskyi/happyfriend/features/settings/ui/SeparateLineItem.kt

```diff
@@ -0,0 +1,26 @@
+package com.yterletskyi.happyfriend.features.settings.ui
+
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import androidx.core.view.forEachIndexed
+import androidx.recyclerview.widget.RecyclerView
+
+class SeparateLineItem : RecyclerView.ItemDecoration() {
+
+    private val paint = Paint().apply {
+        color = Color.rgb(211, 211, 211)
```

Comment:
can we use a color in #RRGGBB format? It would be more meaningful for a developer instead of 211,211,211